auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/UserService.kt

```diff
@@ -148,4 +287,24 @@ class UserService(
             }
         }
     }
+
+    data class DeltaUserDetails(
+        @JsonProperty("id") val id: String, //Not used anywhere yet
```

Comment:
What is id, is it the user's CN?

Reply:
It comes through as blank on the POST request, I am assuming it will become clear when editing users, I will add a TODO to make sure I know what it does (and add a comment/clarify it) by the end of the next part of the ticket if it's ok to be left for now?